### PR TITLE
Semantic updates

### PR DESCRIPTION
**Pull request by marissahuysentruyt**
New semantic html updates throughout portfolio.

-majorly struggled with markup on resume page (potentially means it probably could have been built quite differently)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -54,10 +54,10 @@ <h1>
                 <img src="usedresources/cropped_IMG_2512.png" class="me" alt="Photo of Marissa">
             </div>
                 
-        </div> 
+        </section> 
         
-        <div id="projects">
-            <h2>Projects</h2>
+        <section class="projects">
```
Comment (ITSjwags) [inline index.html:59]: Could probably leave the 'id' here on the section, if we're just using it as a scroll target. No need for the extra 'span' inside the 'h2'
```suggestion
        <section id="projects" class="projects">
```

File: resume/resume.html [hunk 2]
```diff
@@ -30,9 +30,8 @@
         </div> -->
     </header>
 
-    <div class="body-container">
+    <main>
```
Comment (ITSjwags) [inline resume/resume.html:33]: Same comment about leaving the class name on the 'main' element so we can style it the same. Then we can leave the css file as it was!

**Review by ITSjwags — COMMENTED** [review]
Looking real nice so far! Just a couple small comments on my first pass. Let me know if you have any questions about these.

File: contacts/contactpage.html [hunk 3]
```diff
@@ -20,7 +20,7 @@
         </nav>
     </header>
 
-    <div class="body-container">
+    <main>
```
[the remark this answers is not in the document]
Comment (marissahuysentruyt) [reply contacts/contactpage.html:23]: I went through all the files and added .body-container back to the main element. I see I did that for a few of the pages, but didn't for others, so hopefully they're all consistent now.

File: index.html (hunk 1 above)
Comment (marissahuysentruyt) [reply index.html:59]: Honestly, I don't really know why I added that span! But the id is back on the section tag now.

**Review by ITSjwags — APPROVED** [review]
I love it guy

**Pull request merged**